front/src/components/LoginPanel/comp/RegisterForm.js

```diff
@@ -47,31 +62,47 @@ const RegisterForm = (props) => {
     };
 
     return (
-        <form onSubmit={handleSubmit}>
+        <form data-testid="registerForm" onSubmit={handleSubmit}>
             <input
                 type="text"
                 placeholder="Login"
-                className="form-control"
+                className="form-control register-input"
                 value={login}
                 onChange={handleLoginChange}
             />{" "}
             <br />
+            <input
+                type="text"
+                placeholder="Email"
+                className="form-control register-input"
+                value={email}
+                onChange={handleEmailChange}
+            />{" "}
+            <br />
             <input
                 type="password"
                 placeholder="Hasło"
-                className="form-control"
+                className="form-control register-input"
                 value={password1}
                 onChange={handlePassword1Change}
             />{" "}
             <br />
             <input
                 type="password"
                 placeholder="Powtórz hasło"
-                className="form-control"
+                className="form-control register-input"
                 value={password2}
                 onChange={handlePassword2Change}
             />{" "}
             <br />
+            <textarea
+                placeholder="Dlaczego chcesz założyć konto?"
+                className="form-control register-input"
+                rows="5"
+                value={reasonForCreatingAccount}
+                onChange={handlereasonForCreatingAccount}
```

Comment:
When you commit some changes, please make sure that the code actually works. In this example "handleReasonForCreatingAccount" occurs not only in line 44 but 103 as well. Also, spotting this bug is very simple, as page doesn't load and "Failed to compile" error is thrown.